### ISSUE TITLE
[Bug]: Creating index keep failing during rolling upgrade from v2.4.3 to 2.4-20240625-bd8ad106-amd64 with error `The specified key does not exist`

### ISSUE THREAD

**Issue by zhuwenxing** [milvus-io/milvus#34181]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:v2.4.3 to 2.4-20240625-bd8ad106-amd64 
- Deployment mode(standalone or cluster):
- MQ type(rocksmq, pulsar or kafka):    
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

```
[2024-06-25T17:07:27.849Z] [2024-06-25 17:07:24 - ERROR - pymilvus.decorators]: RPC error: [create_index], <MilvusException: (code=1, message=450713269641952208: The specified key does not exist.: key not found[key=files/insert_log/450713269641952153/450713269641952154/450713269641952208/0/450713269641552011];)>, <Time:{'RPC start': '2024-06-25 17:07:23.562610', 'RPC error': '2024-06-25 17:07:24.578649'}> (decorators.py:126)
[2024-06-25T17:07:27.850Z] [2024-06-25 17:07:26 - ERROR - pymilvus.decorators]: RPC error: [create_index], <MilvusException: (code=1, message=450713269641952208: The specified key does not exist.: key not found[key=files/insert_log/450713269641952153/450713269641952154/450713269641952208/0/450713269641552011];)>, <Time:{'RPC start': '2024-06-25 17:07:25.580409', 'RPC error': '2024-06-25 17:07:26.176011'}> (decorators.py:126)
[2024-06-25T17:07:27.851Z] [2024-06-25 17:07:27 - ERROR - pymilvus.decorators]: RPC error: [create_index], <MilvusException: (code=1, message=450713269641952208: The specified key does not exist.: key not found[key=files/insert_log/450713269641952153/450713269641952154/450713269641952208/0/450713269641552011];)>, <Time:{'RPC start': '2024-06-25 17:07:27.210956', 'RPC error': '2024-06-25 17:07:27.724840'}> (decorators.py:126)
[2024-06-25T17:07:29.731Z] [2024-06-25 17:07:29 - ERROR - pymilvus.decorators]: RPC error: [create_index], <MilvusException: (code=1, message=450713269641952208: The specified key does not exist.: key not found[key=files/insert_log/450713269641952153/450713269641952154/450713269641952208/0/450713269641552011];)>, <Time:{'RPC start': '2024-06-25 17:07:28.725705', 'RPC error': '2024-06-25 17:07:29.245850'}> (decorators.py:126)
[2024-06-25T17:07:31.090Z] [2024-06-25 17:07:30 - ERROR - pymilvus.decorators]: RPC error: [create_index], <MilvusException: (code=1, message=450713269641952208: The specified key does not exist.: key not found[key=files/insert_log/450713269641952153/450713269641952154/450713269641952208/0/450713269641552011];)>, <Time:{'RPC start': '2024-06-25 17:07:30.247227', 'RPC error': '2024-06-25 17:07:30.762835'}> (decorators.py:126)
[2024-06-25T17:07:32.460Z] [2024-06-25 17:07:32 - ERROR - pymilvus.decorators]: RPC error: [create_index], <MilvusException: (code=1, message=450713269641952208: The specified key does not exist.: key not found[key=files/insert_log/450713269641952153/450713269641952154/450713269641952208/0/450713269641552011];)>, <Time:{'RPC start': '2024-06-25 17:07:31.763962', 'RPC error': '2024-06-25 17:07:32.276417'}> (decorators.py:126)
[2024-06-25T17:07:33.819Z] [2024-06-25 17:07:33 - ERROR - pymilvus.decorators]: RPC error: [create_index], <MilvusException: (code=1, message=450713269641952208: The specified key does not exist.: key not found[key=files/insert_log/450713269641952153/450713269641952154/450713269641952208/0/450713269641552011];)>, <Time:{'RPC start': '2024-06-25 17:07:33.278168', 'RPC error': '2024-06-25 17:07:33.805994'}> (decorators.py:126)
```

### Milvus Log

failed job: https://qa-jenkins.milvus.io/blue/organizations/jenkins/rolling_update_for_operator_test_simple/detail/rolling_update_for_operator_test_simple/3874/pipeline

log:
[artifacts-kafka-cluster-3874-server-logs.tar.gz](https://github.com/user-attachments/files/15982168/artifacts-kafka-cluster-3874-server-logs.tar.gz)

**Comment by xiaofan-luan** [comment]
/assign @xiaocai2333 
please help on it

**Comment by xiaocai2333** [comment]
This has been fixed by pr: #34133 , please verify it.

**Comment by zhuwenxing** [comment]
verified and fixed in `2.4-20240626-6423b6c7-amd64`
